sklearn/discriminant_analysis.py

```diff
@@ -556,6 +558,8 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
     .. versionchanged:: 0.17
        Deprecated :class:`qda.QDA` have been moved to *QuadraticDiscriminantAnalysis*.
```

Comment:
@raghavrv so you'd like to completely remove the note " Deprecated :class:`qda.QDA` .... " ? 
isn't the purpose to warn users that this previous *QDA* has been changed ?